packages/angular/cli/lib/init.ts

```diff
@@ -20,7 +21,7 @@ const packageJson = require('../package.json');
 
 function _fromPackageJson(cwd = process.cwd()): SemVer | null {
   do {
-    const packageJsonPath = path.join(cwd, 'node_modules/@angular/cli/package.json');
+    const packageJsonPath = resolve.sync('@angular/cli/package.json', { paths: [process.cwd()] });
```

Comment:
Can you remove this change? While this file could use some improvements, it is out of scope for the update related changes in this PR.

Reply:
@clydin so there is a bug here I introduced, thanks for catching it, it should be `paths: [cwd]` instead. 

However, this change **can be** related to this PR. Consider the following scenario:

```
/app/package.json
/app/node_modules/@angular/cli/... <- @angular/cli could be hoisted here by yarn

/app/packages/webapp/package.json <- subproject here
/app/packages/webapp/node_modules/... <- other deps, but @angular/cli not between them
```

Now, if a user runs `ng update` while being `cwd` to `/app/packages/webapp/` the logic above will fail to find the locally installed `@angular/cli` and will use the global one. This can lead to unexpected behavior.

Are you sure you want me to revert it?

Reply:
Please do.  That function will not exist soon.  The problem demonstrated above will be removed as a result as well.